src/hooks/contact/contact.ts

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { FieldValues } from "react-hook-form";
```

Comment:
Typings from abstracted libraries should be avoided in most cases. Consider if this is the case because this type is very wide. Using `FieldValues` from Blocks narrows the FieldValues, but maybe more than you would like. Since it is being used in the submit handler, perhaps the interface for the form default values is a better choice? Your call, just something to think about.